client/components/data/query-site-comments-tree/query-jetpack-site-comments-tree.jsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { requestCommentsTreeForJetpackSite } from 'state/comments/actions';
+
+export class QueryJetpackSiteCommentsTree extends Component {
```

Comment:
Let's rename this to `<QueryCommentsList />` so it's more clear. We'll be confused later if the comment-tree endpoint is available in Jetpack